flink-runtime-web/web-dashboard/src/app/pages/job/dataskew/data-skew.component.html

```diff
@@ -0,0 +1,71 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<nz-card nzType="inner" nzTitle="What is Data Skew?">
+  <p>
+    Your Flink job has data skew when a subset of subtasks in any of the operators receives a
+    disproportionate number of records, potentially overloading a subset of task managers while the
+    rest remain idle, leading to inefficient processing and potentially backpressure and other
+    related problems.
+  </p>
+</nz-card>
+
+<nz-card nzType="inner" nzTitle="Data Skew" [nzLoading]="isLoading" [nzExtra]="extraTemplate">
+  <nz-table
+    class="no-border small"
+    [nzSize]="'small'"
+    [nzData]="listOfVerticesAndSkew"
+    [nzFrontPagination]="false"
+    [nzShowPagination]="false"
+  >
+    <tbody>
+      <tr>
+        <th>Vertex</th>
+        <th>Data Skew Percentage</th>
+      </tr>
+      <ng-container *ngFor="let vertexSkew of listOfVerticesAndSkew">
+        <tr>
+          <td>{{ vertexSkew.vertexName }}</td>
+          <td>{{ vertexSkew.skewPct }}%</td>
+        </tr>
+      </ng-container>
+    </tbody>
+  </nz-table>
+</nz-card>
+
+<nz-card nzType="inner" nzTitle="How is Data Skew Calculated?">
+  <p>
+    Data skew is calculated using
+    <a target="_blank" href="https://en.wikipedia.org/wiki/Coefficient_of_variation">
```

Comment:
I would be wary of linking out to external sites for context. If external linkage appears elsewhere in Flink Dashboard, please consider a language agnostic URL - i.e. https://www.wikipedia.org/wiki/Coefficient_of_variation

Reply:
I will remove the external linkage to be safe.